inst/examples/app.R

```diff
@@ -0,0 +1,38 @@
+library(shiny)
+library(shiny.i18n)
+
+# file with translations
+n18i <- i18n("translation.json")
```

Comment:
can you add an example with csv? it would be very helpful

Reply:
Ok, an example provided as `app_csv.R`.